Makefile

```diff
@@ -56,6 +56,7 @@ lint:
 	  --global module \
 	  --global require \
 	  --global self \
+	  --rule 'indent: [off]' \
```

Comment:
Are we sure we want to do this? `indent` is quite a significant rule, as it applies to every line of code.